esmvaltool/diag_scripts/shared/__init__.py

```diff
@@ -1,8 +1,62 @@
 """Code that is shared between multiple diagnostic scripts."""
 from . import plot
 from ._base import get_cfg, run_diagnostic
+from .python_diag import (
```

Comment:
I think it would be cleaner if you could move all of these names into a `esmvaltool.diag_scripts.shared.names` module. Actually, I think it may be a bit overkill to do it like this, but if you prefer it this way I'm fine with including the names as constants, but please in a separate module.